atlasdb-cassandra/build.gradle

```diff
@@ -56,6 +56,7 @@ dependencies {
   implementation project(':atlasdb-commons')
   implementation project(':commons-annotations')
   implementation project(':commons-executors')
+  implementation project(':timelock-api')
```

Comment:
interesting. so I guess if it's run in a context without the timelock-api project it just falls over at runtime?

Reply:
No, this is needed at compile time:

https://github.com/palantir/atlasdb/blob/64e7944c28a0c04bd7a735b7db820cf4d0dca21f/atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/backup/CqlMetadata.java#L27